### ISSUE TITLE
Couldn't find env file!

### ISSUE THREAD

**Issue by mohammadmahboubeh** [shihjay2/docker-nosh#2]
Pleasure day!
I got this error message when trying to nosh running:
![Capture](https://user-images.githubusercontent.com/46927354/103139878-07186000-46f2-11eb-8492-ad4ed22e7b2b.PNG)

**Comment by mohammadmahboubeh** [comment]
Trying to run docker-nosh_webserver1:

Please advise,

![Capture](https://user-images.githubusercontent.com/46927354/103150719-a50a3a00-4787-11eb-82e2-527608219deb.PNG)

**Comment by shihjay2** [comment]
Can you verify that the env file is indeed missing in the docker-nosh/nosh directory?  If you did a git clone as instructed, the file should have been included (as seen [here](https://github.com/shihjay2/docker-nosh/tree/master/nosh)); if it is indeed missing, it might be something related to Powershell but since I'm not a Windows user, it's only a guess.

**Comment by mohammadmahboubeh** [comment]
For env file I solved it. But when I finished the installation and looking to docker webserver is trying to start. Kindly see the pervious picture.

Thanks

**Comment by shihjay2** [comment]
See update on commit 4e1b6bca713b4966202c9298473d5b2a504a6f0b (you can restart the entire install routine or see the changes in the commit on your nginx.conf file and then restart the webserver.

**Comment by mohammadmahboubeh** [comment]
Thanks I will do and let you know...

**Comment by mohammadmahboubeh** [comment]
Now the webserver is working! 
How can get in the the nosh web application?
![Capture](https://user-images.githubusercontent.com/46927354/103152617-cb85a080-479a-11eb-9b1c-ef3058116d00.PNG)

**Comment by mohammadmahboubeh** [comment]
I tried localhost!! 
![Capture](https://user-images.githubusercontent.com/46927354/103152677-62eaf380-479b-11eb-9041-cae6252ab19b.PNG)

**Comment by shihjay2** [comment]
Looks like you're missing the ./.nosh_app_key (generated in the init_win.sh); confirm that it's in the root directory.

**Comment by mohammadmahboubeh** [comment]
I think it's missing...
![Capture](https://user-images.githubusercontent.com/46927354/103153301-a3993b80-47a0-11eb-84e9-cf1dfbf3b9c7.PNG)

**Comment by mohammadmahboubeh** [comment]
By the way I am using Ubuntu inside windows so I used ./init.sh

**Comment by shihjay2** [comment]
You need to use ls -a since it is a dot file.

**Comment by mohammadmahboubeh** [comment]
Here you are
![Capture](https://user-images.githubusercontent.com/46927354/103153414-79944900-47a1-11eb-89f4-0783fe02e929.PNG)

**Comment by shihjay2** [comment]
What is the content of the file (sudo cat .nosh_app_key)?  Is it empty?

**Comment by mohammadmahboubeh** [comment]
![Capture](https://user-images.githubusercontent.com/46927354/103153775-fd4f3500-47a3-11eb-86a5-e0eab6b85e91.PNG)

**Comment by mohammadmahboubeh** [comment]
I tried to install it again and I got this...
![Capture](https://user-images.githubusercontent.com/46927354/103153802-32f41e00-47a4-11eb-9845-f7638c576f0b.PNG)

**Comment by shihjay2** [comment]
You might want to reboot your ubuntu VM; I can't tell if you didn't shut down docker when you re-installed it.

**Comment by mohammadmahboubeh** [comment]
I just reboot the Ubuntu and the same problem! Should I shutdown docker before re-installing the nosh?

**Comment by shihjay2** [comment]
Yes, you need to do docker-compose down.  I would also delete your root
folder in case there is any stray or incorrectly formatted files and start
over with git clone.

**Comment by mohammadmahboubeh** [comment]
After installation I typed docker-compose up and got this:
![Capture](https://user-images.githubusercontent.com/46927354/103154969-9f731b00-47ac-11eb-808a-48b1097b5d3c.PNG)

**Comment by shihjay2** [comment]
Identified a problem with the mariadb docker image (unable to create InnoDB database) and also identified a problem with the init script (that is why you didn't have an app key).  These will be fixed soon.

**Comment by shihjay2** [comment]
I think I figured out the problem - the database volume was already present and encrypted and when you created a new install - there was mismatch hence the database error.  To restart (will wipe all existing data) use the command `docker-compose down -v`, then use the `./init.sh` command; updated the init scripts to fix the problem with key generation in commit 46dea2fe08d0853d5f31ef5f8b7a5662f1d3a730
